src/dividend/IDividendDistributor.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+interface IDividendDistributor {
```

Comment:
Polish this. Move method comments here and have DividendDistributor inherit and implement the interface.